src/NuGet.config

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
-    <add key="ChazShare" value="\\cmb-pbi2\Public\Nugets" />
+    <add key="private-nuget-package" value="C:\git\nugetpackages" />
```

Comment:
I don't think we need to check-in this change.

Reply:
I'm OK with it as its not pointing to my share anymore and always local